gnat2goto/driver/tree_walk.adb

```diff
@@ -1189,7 +1190,9 @@ package body Tree_Walk is
       Set_Source_Location (Ret, Sloc (N));
       Set_Type (Ret, Constant_Type);
       Set_Value (Ret,
-                 Convert_Uint_To_Binary (Intval (N), Pos (Constant_Width)));
+                 Convert_Binary_To_Hex
```

Comment:
Is there any value in adding a `Convert_Uint_To_Hex` function to do this 'directly' - even if it's just to make the code more readable?

Reply:
Should be easy enough to do